modules/banners/README.rdoc

```diff
@@ -0,0 +1,7 @@
+= Banners
+The Banners API provides a means to generate VA.gov banners quickly, without relying on content-build.
+
+== Installation
+Ensure the following line is in the root project's Gemfile:
+
+  gem 'banners', path: 'modules/banners'
```

Comment:
Do we need the path portion here? I noticed it was not necessary in the root Gemfile

Reply:
Hmm, probably not, removed.